markets/legacy-market/cannonfile.toml

```diff
@@ -139,22 +132,20 @@ args = [
     #["0x7dA35c7eE680A1e81eFCb2e9edD0c8D039D5211e"]
 ]
 fromCall.func = "owner"
-depends = ["invoke.upgradeProxy"]
 
 [invoke.authorizedBroker]
 target = ["v2x.SynthetixDebtShare"]
 func = "addAuthorizedBroker"
 args = ["<%= contracts.Proxy.address %>"]
 fromCall.func = "owner"
-depends = ["invoke.upgradeProxy"]
 
 # create spartan council pool
 [invoke.createPool]
 target = ["v3.CoreProxy"]
 from = "<%= settings.pool_owner %>"
 func = "createPool"
 args = ["<%= settings.sc_pool_id %>", "<%= settings.pool_owner %>"]
-depends = ["invoke.setFeatureFlagPool"]
+depends = ['invoke.setFeatureFlagPool']
```

Comment:
create pool will fail if the `createPool` feature flag is not enabled, which is done in the `invoke.setFeatureFlagPool` step. Since `invoke.setFeatureFlagPool` outputs are not used as arguments/interpolated parrameters for this step (but still used in contract storage), it must be manually depended upon